src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TagHelperAttribute<TValue> where TValue : class
```

Comment:
Ahh :+1: was thinking of combining `Minimized` into `Value` by being `null` which is where `: class` came into play but realized that's not possible since `TagHelperAttribute<object>` is used for `TagHelperContext.AllAttributes`.